pages/science/index.rst

```diff
@@ -37,56 +36,55 @@ technology applications.
 
          .. container::
             :tagname: a
-            :attributes: href=phases.html
-                         title=Phases
+            :attributes: href=thermodynamics.html
+                         title=Thermodynamics
 
             .. container:: card-header section-card
                :tagname: div
 
-               Phases
+               Thermodynamics
 
          .. container:: card-body
 
             .. container:: card-text
 
-               The theory behind some of Cantera's phase models.
+               The theory behind how Cantera calculates species and phase thermodynamic properties.
 
       .. container:: card
 
          .. container::
             :tagname: a
-            :attributes: href=science-species.html
-                         title=Species
+            :attributes: href=reactions.html
+                         title=Reactions
```

Comment:
```suggestion
            :attributes: href=kinetics.html
                         title=Kinetics
```

Reply:
Could I suggest "Kinetics and Reaction Rates"? That keeps the breadth, but also hints at the fact that much of the description to follow will be about the different reaction rate parameterizations available.

Reply:
I like titling the card "Kinetics and Reactions Rates," but switching the html tag to the shorter "kinetics" feels like the right move, to me...